app/components/EventDetail/index.js

```diff
@@ -46,58 +47,78 @@ class EventDetail extends Component {
     return (
       <div className={styles.eventDetail}>
         {this.state.isEditing && (
-          <Lightbox>
+          <Lightbox
+            toggleLightbox={() => {
+              this.toggleEditing()
+            }}
+          >
             <EventEdit
               event={event}
-              toggleEditing={() => {
-                this.toggleEditing()
-              }}
+              toggleLightbox={() => this.toggleEditing()}
             />
           </Lightbox>
         )}
         <div className={styles.head}>
-          <h2>
-            <FormattedMessage {...messages.event} />: {event.companyName} -{' '}
-            {event.date && event.date.toString()}
-          </h2>
+          <h2>{event.companyName}</h2>
           {hasEventPermission(user) && (
             <div className={styles.buttonRow}>
-              <Button color='bright' onClick={() => this.toggleEditing()}>
-                Edit
+              <Button
+                color='bright'
+                onClick={() => this.toggleEditing()}
+                rounded={true}
+              >
+                <FormattedMessage {...messages.edit} />
               </Button>
-              <Button color='danger' onClick={() => onRemoveEvent(event.id)}>
-                Delete
+              <Button
+                color='danger'
+                onClick={() => onRemoveEvent(event.id)}
+                rounded={true}
+              >
+                <FormattedMessage {...messages.delete} />
               </Button>
             </div>
           )}
         </div>
         <div className={styles.info}>
+          {event && event.responsible && (
+            <div>
+              <h4>Responsible</h4>
+              <div>{event.responsible}</div>
+            </div>
+          )}
           <div>
-            <h4>Company</h4>
-            <div>{event.companyName}</div>
+            <h4>
+              <FormattedMessage {...messages.date} />
+            </h4>
+            <div>
+              {event.date && moment(event.date).format('MMM DD, HH:mm')}
+            </div>
           </div>
           <div>
-            <h4>Location</h4>
+            <h4>
+              <FormattedMessage {...messages.location} />
+            </h4>
             <a
-              href={`https://www.google.com/maps/search/?api=1&query=${encodeURIComponent(
-                event.location
-              )}`}
+              href={`https://www.google.com/maps/search/?api=1&query=
```

Comment:
What happened here? Prettier?

Reply:
Nope, the "line length exceeded warning". Should I change it back?

Reply:
Nah, if it works it's fine.